example/ck_tile/03_gemm/gemm_basic.cpp

```diff
@@ -12,7 +12,8 @@
 #include "ck_tile/host.hpp"
 #include "gemm_basic.hpp"
 
-template <typename ALayout, typename BLayout, typename CLayout>
+template <typename ADataType, typename BDataType, typename AccDataType, typename CDataType,
```

Comment:
Btw, are ALayot, Blayout CLayout unused?

Reply:
They are used on lines 70 and 71.